apps/central-scan/frontend/src/screens/admin_actions_screen.tsx

```diff
@@ -94,28 +110,10 @@ export function AdminActionsScreen({
     setIsBackingUp(false);
   }, [backup, logger, userRole]);
 
-  const deleteBallotData = useCallback(async () => {
-    toggleIsConfirmingZero();
-    setIsDeletingBallotData(true);
-    await zeroData();
-    setIsDeletingBallotData(false);
-  }, [zeroData]);
+  const [isSetMarkThresholdModalOpen, setIsMarkThresholdModalOpen] =
+    useState(false);
 
-  useEffect(() => {
-    if (isFactoryResetting) {
-      let isMounted = true;
-      void (async () => {
-        usbDriveEject(userRole);
-        await unconfigureServer();
-        if (isMounted) {
-          setIsFactoryResetting(false);
-        }
-      })();
-      return () => {
-        isMounted = false;
-      };
-    }
-  }, [isFactoryResetting, unconfigureServer, usbDriveEject, userRole]);
+  const markThresholdOverrides = markThresholdOverridesQuery.data ?? undefined;
```

Comment:
Originally I had "Loading Admin Actions..." show but I didn't like the whole UI jumpiness. In 99% of cases there will not be override data and the default will be accurate.

The `?? undefined` is because it might be `null` coming from the backend and I just like to convert at the boundaries for consistency.

Reply:
Ok, thanks for explaining!